seed.js

```diff
@@ -1,15 +1,15 @@
 // This file allows us to seed our application with data
 // simply run: `node seed.js` from the root of this project folder.
 
-// var db = require('./models');
+var db = require('./models');
 
-// var new_campsite = {description: "Sharp rocks. Middle of nowhere."}
+var new_skatepark = {description: "Sharp rocks. Middle of nowhere."}
```

Comment:
It seems like you didn't actually edit your seed file to be relevant to the structure of your skatespots.